dev/tests/js/jasmine/tests/app/code/Magento/Ui/base/js/grid/toolbar.test.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+/*eslint max-nested-callbacks: 0*/
+define([
+    'Magento_Ui/js/grid/toolbar',
+    'jquery'
+], function (Toolbar, $) {
+    'use strict';
+
+    describe('Magento_Ui/js/grid/toolbar', function () {
+        var toolbarObj = new Toolbar({
+                index: 'listing_top',
+                dataScope: '',
+                columnsProvider: 'magento',
+                provider: 'provider',
+                name: 'magento'
+            }),
+            toolbarType,
+            originToolbar;
+
+        beforeEach(function () {
+            originToolbar = toolbarObj;
```

Comment:
Could you please create new toolbar Obj before each test ? and i mean `spyOn($, '_data')` might be reverted to the original state, to not affect other tests